.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "c-blosc"]
 	path = c-blosc
 	url = https://github.com/Blosc/c-blosc.git
+[submodule "zfp"]
+	path = zfp
+	url = https://github.com/LLNL/zfp.git
```

Comment:
We shouldn't need the source code here since we can go the dependency route

```suggestion
```

Also please remove the submodule as well.